### DISCUSSION TITLE
Welcome to Zimodem Discussions!

### DISCUSSION THREAD

**Discussion by bozimmerman** [bozimmerman/Zimodem#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lycamp** [comment]
Hi. I am a retro enthusiast. My handle on most boards and forums is Desert-Fox or Lycamp. I run a Color64 BBS called The Oasis.
I'm currently working a a BBS project for the Commander X16 and am hoping to find a place where I can go with questions about Zimodem.
I'm a an old retired IT guy from Verizon with time on my hands and live in Chesapeake Va.
